pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -1685,7 +1685,14 @@
     private double managedLedgerCacheEvictionWatermark = 0.9;
     @FieldContext(category = CATEGORY_STORAGE_ML,
             doc = "Configure the cache eviction frequency for the managed ledger cache. Default is 100/s")
+    @Deprecated
     private double managedLedgerCacheEvictionFrequency = 100.0;
+
+    @FieldContext(category = CATEGORY_STORAGE_ML,
+            doc = "Configure the cache eviction interval for the managed ledger cache. Must >= 1 ms, "
+                    + " and default is 10 ms")
+    private long managedLedgerCacheEvictionIntervalMs = 10;
+
```

Comment:
Need to consider more about the compatibility

The user might use the old one which means using a different value as the default value, but after upgrading to the new version, the default value will be applied, but we should apply the configured value  by users.

Reply:
You're right, I'll add compatible logic here.